dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/domain/event/LogEventSerializer.kt

```diff
@@ -25,10 +25,9 @@ internal class LogEventSerializer(
         val sanitizedTags = dataConstraints
             .validateTags(log.ddtags.split(","))
             .joinToString(",")
-        // TODO: 13/04/2021 RUMM-1298 Handle the RESERVED_ATTRIBUTES in the `toJson` method
         val sanitizedAttributes = dataConstraints
             .validateAttributes(log.additionalProperties)
-            .filterKeys { it.isNotBlank() && it !in RESERVED_PROPERTIES }
```

Comment:
nice one, I think you can mark that task as done ;)

Reply:
it was already marked as `DONE` by you :)

Reply:
My bad then :(. I guess I forgot to remove that code when I replaced that method. Thanks for taking care of it.